### PR TITLE
Add phpunit

### PR DESCRIPTION
**Pull request by tarnawski**

### CODE REVIEW

File: phpunit.xml.dist [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:noNamespaceSchemaLocation="http://schema.phpunit.de/6.5/phpunit.xsd"
```
Comment (dominikhajduk) [inline phpunit.xml.dist]: shouldn't it be `6.5` -> `9.0`?

File: phpunit.xml.dist [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+         backupGlobals="false"
+         colors="true"
+         verbose="true">
+
+    <testsuites>
+        <testsuite name="unit">
+            <directory>tests/unit</directory>
+        </testsuite>
+    </testsuites>
+</phpunit>
```
Comment (dominikhajduk) [inline phpunit.xml.dist]: missing empty line

File: phpunit.xml.dist (hunk 1 above)
Comment (tarnawski) [reply phpunit.xml.dist]: The second approach looks much better IMHO

Comment (dominikhajduk) [reply phpunit.xml.dist]: It's also default for phpunit I think.

File: phpunit.xml.dist [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:noNamespaceSchemaLocation="vendor/phpunit/phpunit/phpunit.xsd"
+         backupGlobals="false"
+         colors="true"
+         verbose="true">
```
Comment (mateuszmaslanka) [inline phpunit.xml.dist]: I think the closing `>` can be moved to new line.

Comment (tarnawski) [reply phpunit.xml.dist]: Can be moved... but there is any reason to do that? :thinking:

Comment (mateuszmaslanka) [reply phpunit.xml.dist]: Similar like `,` after the last element in PHP array - if you want to add/remove the last element you don't need to care about `>`

**Pull request merged**